### DISCUSSION TITLE
Toolbar

### DISCUSSION THREAD

**Discussion by francoisschwarzentruber** [tableaunoir/tableaunoir#154] [Ideas]
In this discussion, we may discuss how to improve the toolbar.

**Comment by deleporte** [comment]
**Separate navigation bar**
Pasting the comment from #110 :
Controls for the vertical separator, going left/right, and showing the whole screen, may be placed at the bottom right of the screen. They don't exactly have the same status as drawing/selecting tools.
![image](https://user-images.githubusercontent.com/12908293/115738367-48add480-a38d-11eb-9303-520fc1f4496d.png)

**Hiding and showing toolbar(s)**
An inexperienced user may accidentally press H (say, after clicking outside of a text box) and hide the toolbar without a clue about how to activate it again. Solutions include : 
1. displaying a text ("fullscreen mode; show toolbar by pressing H") which disappears after a few seconds, 
2. pressing ESC while the toolbar is hidden shows it again (intuitive escape attempt from fullscreen mode) rather than showing the menu
3. keeping one button visible to unfold the toolbar.

**Reply by francoisschwarzentruber** [reply]
Dear @deleporte,

Thank you for all your suggestions.

**Separate navigation bar**
Although I did not add a new navigation bar, I reorganised a bit the toolbar:
- menu
- edition
- undo/redo
- divide the screen (at the middle of the screen)
- navigation in the board or in time
- share button

At the bottom, you have the animation bar when it is visible.

**Hiding and showing toolbar(s)**
I agree that some people may be very sad after accidentally pressing H. So the options you propose are very nice.
1. I like it. It is now implemented. Hoping it will not annoy people.
2. Yes, why not. But I find it convenient to still to be able to go in the menu normally. Maybe I am wrong.
3. Some people really need that **nothing** is shown except the board. So it is not a good idea for them.
